.github/workflows/mutest-issue-generate.yml

```diff
@@ -0,0 +1,50 @@
+name: Generate Mutation Test Errors 
+
+on:
+  # for testing 
+  push:
+    branches:
+      - mut-test-gen
```

Comment:
Why not off of `main`?

Reply:
just so that it runs every time i push to my branch. 

All of it is just for testing